src/platform/qt/Window.cpp

```diff
@@ -1632,6 +1632,11 @@ void Window::setupMenu(QMenuBar* menubar) {
 	addGameAction(tr("&Cheats..."), "cheatsWindow", openControllerTView<CheatsView>(), "tools");
 #ifdef ENABLE_SCRIPTING
 	m_actions.addAction(tr("Scripting..."), "scripting", this, &Window::scriptingOpen, "tools");
+	
+	addGameAction(tr("Load recent script"), "loadscript", [this] {
+		m_mruFiles = m_config->getMRU(ConfigController::MRU::Script);
```

Comment:
This overwrites the member variable `m_mruFiles`, which is _not_ what you want.

Reply:
I changed the location of the shortcut to appear in the scripting window but was not able to bind it to any keys. It does not overwrite m_mruFiles and does play the most recently run script in the scripting window